flags.json

```diff
@@ -1,47 +1,38 @@
 [
     {
        "ID": 0,
-       "Flag": "yctf-5f39410913126cea50a3e6fcdb7312628a869da376985fdb7ef7d0a41254a0e3", 
```

Comment:
In the flags branch, my idea is to completely getting rid of the flags.json, while the file is there it's not used any longer

Reply:
One thing to think about here is that the flags.json binds an identifier with its template. I don't think I saw an order enforced in the flags branch, but I know it's a WIP. In my mind, I think having some degree of configuration whereby you bind a specific template with a specific ID is okay.